<launch>
	<!-- Parameters -->
	<param name="minimum_disparity" value="-32" />
	<param name="WIDTH" value="1920" />
	<param name="HEIGHT" value="1080" />

        <!-- ROS Serial Node -->
	<node pkg="rosserial_python" type="serial_node.py" name="ArduinoConnection" args="/dev/ttyACM0"></node>

	<!-- Gimbal Control -->
	<node pkg="project" type="gimbal_control.py" name="GimbalControl"></node>

	<!-- Capture Images -->
	<node pkg="project" type="captureImages.py" name="captureImages"></node>

	<!-- Rectify Images -->
	<node pkg="project" type="rectifyImages.py" name="rectifyImages"></node>
	
	<!-- Create Disparity -->
	<node pkg="project" type="createDisparityMap.py" name="createDisparity"></node>

	<!-- Create Point Cloud -->
	<node pkg="project" type="computePointCloud.py" name="computePointCloud"></node>

</launch>
